_episodes/05b-advanced-containers.md

```diff
@@ -0,0 +1,315 @@
+---
+title: "Creating More Complex Container Images"
+teaching: 30
+exercises: 30
+questions:
+- "How can I make more complex container images? "
+objectives:
+- "Explain how you can include files within Docker images when you build them."
+- "Explain how you can access files on the Docker host from your Docker containers."
+keypoints:
+- You can include files from your Docker host into your Docker images by using the `COPY` instruction in your `Dockerfile`.
+- Docker allows containers to read and write files from the Docker host.
+---
+
+In order to create and use your own containers, you may need more information than
+our previous example. You may want to use files from outside the container, copy
+those files into the container, and just generally learn a little bit about software
+installation. This episode will cover these. Note that the examples will get gradually
+more and more complex - most day-to-day use of containers can be accomplished
+using the first 1-2 sections on this page.
+
+## Using scripts and files from outside the container
+
+In your shell, change to the `sum` folder in the `docker-intro` folder.
+~~~
+$ cd ~/Desktop/docker-intro/sum
```

Comment:
maybe `wget` these? or the whole folder when it is first introduced?

Reply:
maybe `ls` too? to see the filenames.

Reply:
Added `ls`, not sure about `wget`.

Reply:
I think `wget` is only needed if you want to download the docker-intro files you mention.  You can do it with clicking download instead.